docker-compose/docker-compose-dse7.yml

```diff
@@ -0,0 +1,53 @@
+version: '2'
+services:
+  dse-7:
+    image: datastax/dse-server:7.0.0-alpha.3
```

Comment:
Should the tag come from `.env` file (like `DSE7_TAG`)?

Reply:
Sorry, forgot to remove the image hardcode for testing purposes. I've updated to use the reference `DSE7TAG` which is the latest DSE7 image tag in Docker Hub. The command is `DSE7TAG="$(curl -s https://registry.hub.docker.com/v2/repositories/datastax/dse-server/tags/ | jq -r '[.results[] | select(.name | startswith("7.0.0"))] | sort_by(.last_updated) | last | .name')"` in `start_dse7.sh`.